src/world/world_server.cpp

```diff
@@ -21,12 +21,13 @@ along with this program.  If not, see http://www.gnu.org/licenses/
 #include "world_server.h"
 
 #include "common/application.h"
-#include "common/console_service.h"
 #include "common/logging.h"
+#include "common/vana_time.h"
 
 namespace
 {
-    std::unique_ptr<SqlConnection> sql;
+    std::unique_ptr<SqlConnection>  sql;
```

Comment:
World server itself isn't doing any SQL stuff, does it need its own sql handle?

Reply:
its used in this `UpdateDailyTallyPoints` method which I moved back outside per request (since it doesnt belong to conquest system)